packages/react/package.json

```diff
@@ -49,7 +49,15 @@
     "test:story": "yarn test:engines && cross-env TZ=America/Chicago jest src/components/StorybookSnapshots.test.js --no-watchman --colors",
     "test:clear": "yarn test:engines && cross-env TZ=America/Chicago jest --no-watchman --clearCache --colors",
     "test:engines": "check-node-version --node $(cat ../../.nvmrc) --yarn 1.x",
-    "test:ci": "yarn test --ci --logHeapUsage --coverage"
+    "test:ci": "yarn test --ci --logHeapUsage --coverage",
+    "test:e2e:update": "cypress-image-diff -u",
+    "test:e2e": "cross-env BABEL_ENV='e2e' NODE_ENV='e2e' cypress run-ct --browser chrome --config video=false,screenshotOnRunFailure=false",
+    "test:e2e:interactive": "cross-env BABEL_ENV='e2e' NODE_ENV='e2e' cypress open-ct --browser chrome --config video=false,screenshotOnRunFailure=false",
+    "test:e2e:ci": "cross-env BABEL_ENV='e2e' NODE_ENV='e2e' cypress run-ct  --browser chrome --headless --config --record video=false,screenshotOnRunFailure=false",
+    "reports:clean": "rimraf .nyc_output reports coverage",
```

Comment:
That may be a good idea. We were not cleaning our jest coverage folder before. I may want to leave off the coverage directory actually. This script will run on pre-commit. I can see devs wanting to commit some code and still see the coverage report to keep working without having to run all the test again. I had to include the `.nyc_output` because a bug in `nyc` will cause an error if the folder if present before test are ran. I also think reports can be deleted since it's also a temp file.

So I will remove coverage from this script and just have the temp folders get cleaned up. cool?

Reply:
Sounds good to me.